code/game/machinery/recycle_vendor.dm

```diff
@@ -414,6 +441,8 @@
 			return TRUE
 	update_icon()
 
+#define MINIMUM_BUDGET 800 // tax purposes
```

Comment:
this is the Minimum Budget Value, below which it is impossible for any source to collect money from the budget without giving the AME an influx of materials to compensate.
This is necessary for the operation of the system. Currently set to 800.